src/battle-animations.ts

```diff
@@ -1822,16 +1821,16 @@ class PokemonSprite extends Sprite {
 	}
 
 	behindx(offset: number) {
-		return this.x + (this.isBackSprite ? -1 : 1) * offset;
+		return this.x + (this.isFrontSprite ? 1 : -1) * offset;
 	}
 	behindy(offset: number) {
-		return this.y + (this.isBackSprite ? 1 : -1) * offset;
+		return this.y + (this.isFrontSprite ? 1 : -1) * offset;
```

Comment:
Whoops, looks like I missed something. I'll fix this real quick and PR. Looking over the rest, I think this is the only mistake.